### ISSUE TITLE
SuspendFunWithCoroutineScopeReceiver: Add alias SuspendFunctionOnCoroutineScope

### ISSUE THREAD

**Issue by osipxd** [detekt/detekt#6087]
## Expected Behavior of the rule

```kotlin
@Suppress("SuspendFunctionOnCoroutineScope")
public suspend fun CoroutineScope.joinChildrenCoroutines() {
    coroutineContext.job.joinChildren()
}
```

`SuspendFunWithCoroutineScopeReceiver` should not report any issues because `SuspendFunctionOnCoroutineScope` is suppressed.

## Context

Similar to rules mentioned in #5540, IDEA has [inspection SuspendFunctionOnCoroutineScope](https://github.com/JetBrains/intellij-community/blob/idea/231.9011.14/plugins/kotlin/idea/src/org/jetbrains/kotlin/idea/inspections/coroutines/SuspendFunctionOnCoroutineScopeInspection.kt) working similar to `SuspendFunWithCoroutineScopeReceiver`, so it would be great to have the possibility to suppress both inspections with a single `Suppress`.

<img width="734" alt="image" src="https://github.com/detekt/detekt/assets/3684355/2241a3a8-6d3f-4096-a237-525b1430e4c9">

**Comment by BraisGabin** [comment]
That have a lot of sense and it should be fairly easy to do. This is up to grab by anyone. @osipxd do you want to give it a try?

**Comment by osipxd** [comment]
> @osipxd do you want to give it a try?

Yes, I'll make PR

**Comment by osipxd** [comment]
Done
